controllers/schedule/gc/controller.go

```diff
@@ -90,13 +92,9 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 				if !finished {
 					if untilStop != 0 {
-						if requeuAfter == 0 || requeuAfter > untilStop {
+						if untilStop>requeuAfter {
 							requeuAfter = untilStop
 						}
-
-						r.Recorder.Event(schedule, recorder.ScheduleSkipRemoveHistory{
-							RunningName: innerObj.GetName(),
-						})
```

Comment:
after we removed these lines to emit event, do we still need to change `requeueAfter` to 10 mins ?

Reply:
It is not necessary. I will reset it.

Reply:
Updated